docs/DafnyRef/UserGuide.md

```diff
@@ -218,27 +210,35 @@ Various options control the verification process, in addition to all those descr
    - `--boogie-filter`
 
 
-#### 25.5.1.4. `dafny translate` {#sec-dafny-translate}
+#### 25.5.1.4. `dafny translate <platform>` {#sec-dafny-translate}
```

Comment:
I think language is more correct than platform, because we translate to a specific language such as Java or (potentially) Scala, as opposed to directly translating to something that is consumed by a runtime/platform like Java bytecode.

Reply:
Agreed

Reply:
:rocket: